jlm/llvm/opt/alias-analyses/Steensgaard.cpp

```diff
@@ -93,18 +46,31 @@ class SteensgaardPointsToGraphConstructionStatistics final : public jlm::util::S
   {}
 
   void
-  Start(const LocationSet & locationSet)
+  StartAliasAnalysisStatistics(const jlm::rvsdg::graph & graph) noexcept
```

Comment:
Would StartSteensgardsStatistics be more informative

Reply:
Or is this one step in the pipeline that all together is Steensgard?

Reply:
Renamed.